docs/examples/move/ctf/challenge_3/solution_step_2.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Steps 2–5: Merge coins, split them, call get_flag, then transfer both coins
+
+# Load environment variables
+source ../.env
+
+# 🧩 Replace these with actual coin object IDs from Step 1
+COIN_1_ID=<coin-1-object-id>
+COIN_2_ID=<coin-2-object-id>
+COIN_3_ID=<coin-3-object-id>
+
+# 🧩 Replace this with your address (use: iota client addresses)
+CALLER_ADDRESS=<caller-address>
+
+iota client ptb \
+  --assign coin_1 @"$COIN_1_ID" \
+  --assign coin_2 @"$COIN_2_ID" \
+  --assign coin_3 @"$COIN_3_ID" \
+  --merge-coins coin_1 [coin_2, coin_3] \
+  --split-coins coin_1 [5,1] \
```

Comment:
I don't really see the benefit of assigning the values here since the env variables are already very similar, but feel free to resolve and don't change this
```suggestion
  --merge-coins @"$COIN_1_ID" [@"$COIN_2_ID", @"$COIN_3_ID"] \
  --split-coins @"$COIN_1_ID" [5,1] \
```

Reply:
Makes sense, thanks for the suggestion 👍